static/styles/typography.less

```diff
@@ -98,6 +98,7 @@ strong, b {
 }
 em,cite{
 	font-style:italic;
+	font-weight:600;
```

Comment:
I believe this should be `400` to match the text around it. If someone wants the text to be bold, they should use a `<strong>` element as well.